src/lib.rs

```diff
@@ -689,6 +689,7 @@
 extern crate nb;
 extern crate void;
 
+pub mod analog;
```

Comment:
(Please note that I haven't worked with a DAC so far, so the following concerns may be unfounded. It would be great if someone with more experience in this area could pitch in.)

I'm concerned about the naming, and how well it will suit our needs once we add DAC traits. While `analog` lines up nicely with `digital`, having a single `analog` module means both ADC and DAC will have to share the same namespace, and you're already reserving some attractive names for ADC.

Digital I/O is usually implemented in the same peripheral, and some drivers need to concern themselves with both input and output on the same pin. For that reason, I think it's a good idea to have a single `digital` module. The question is, is the same true for ADC and DAC?

If the answer is "no", then I suggest renaming the module to `adc`.

Reply:
My experience lines up with your intuition here -- a rename to `adc` would be preferable IMO.

It's even what I started typing when bikeshedding the trait name, before I double checked.

Reply:
`adc` and `Once` works for me.